js/adapt-assessmentArticleModel.js

```diff
@@ -514,7 +514,14 @@ define([
             function step() {
                 for (var j = 0, count=Math.min(syncIterations, len-i); j < count; i++, j++) {
                     var question = qs[i];
-                    question.reset(assessmentConfig._questions._resetType, true);
+                    
+                    if(assessmentConfig._questions._resetIncorrectOnly) {
+                        if (!question.get('_isCorrect')) {
+                            question.reset(assessmentConfig._questions._resetType, true);
+                        }
+                    } else {
+                        question.reset(assessmentConfig._questions._resetType, true);
+                    }
```

Comment:
Since there's 2 identical lines in here I think this could be simplified. In line with my suggested property name change above, how about:
```
if (!question.get('_isCorrect') || assessmentConfig._questions._canResetCorrectAnswers !== false) {
  question.reset(assessmentConfig._questions._resetType, true);
}
```

Which should also maintain backward compatibility

Reply:
my thinking was to have the property name 'logic' reversed e.g. `dontResetCorrect` (not the best wording but you get the idea) then you could do something like:
```js
if (assessmentConfig.dontResetCorrect && question.get('_isCorrect')) continue;
```

the setting should also be moved up a level as the `_questions` object is all about overriding the properties that belong to the question components